[2025-10-07T10:16:44.810+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T10:16:44.964+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=3 [queued]>
[2025-10-07T10:16:45.014+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=3 [queued]>
[2025-10-07T10:16:45.016+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-10-07T10:16:45.132+0000] {taskinstance.py:2890} INFO - Executing <Mapped(_PythonDecoratedOperator): process_batch> on 2025-10-07 10:01:57.519057+00:00
[2025-10-07T10:16:45.391+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'vnstock_weekly_etl_incremental', 'process_batch', 'manual__2025-10-07T10:01:57.519057+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpv64rcn5a', '--map-index', '3']
[2025-10-07T10:16:45.400+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask process_batch
[2025-10-07T10:16:45.386+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=292) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T10:16:45.411+0000] {standard_task_runner.py:72} INFO - Started process 504 to run task
[2025-10-07T10:16:45.728+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-07T10:16:49.632+0000] {task_command.py:467} INFO - Running <TaskInstance: vnstock_weekly_etl_incremental.process_batch manual__2025-10-07T10:01:57.519057+00:00 map_index=3 [running]> on host e3278e0f0bbc
[2025-10-07T10:16:52.170+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='vnstock_weekly_etl_incremental' AIRFLOW_CTX_TASK_ID='process_batch' AIRFLOW_CTX_EXECUTION_DATE='2025-10-07T10:01:57.519057+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-07T10:01:57.519057+00:00'
[2025-10-07T10:16:52.201+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-07T10:16:52.205+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VES: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.211+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VEF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.216+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VEC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.222+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VEA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.223+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE9: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.224+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE8: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.225+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE4: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.226+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.226+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.227+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VE1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.230+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.231+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.231+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.232+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDN: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.232+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.233+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDL: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.233+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.234+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VDB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.234+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCX: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.234+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCW: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.237+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.239+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCS: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.240+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCR: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.240+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCP: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.241+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCM: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.241+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCI: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.242+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.243+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.244+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCE: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.246+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.247+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.248+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VCA: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.249+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC9: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.250+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC7: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.252+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC6: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.253+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC5: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.253+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC3: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.256+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC2: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.257+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VC1: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.258+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VBH: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.260+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VBG: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.265+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VBC: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.266+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VBB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.268+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VAV: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.269+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VAT: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.270+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VAF: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.271+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý VAB: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.271+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý V21: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.271+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý V15: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.274+0000] {logging_mixin.py:190} INFO - ⚠️ Lỗi khi xử lý V12: 'Limiter' object has no attribute 'ratelimit'
[2025-10-07T10:16:52.276+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T10:16:52.304+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-07T10:16:52.306+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=vnstock_weekly_etl_incremental, task_id=process_batch, run_id=manual__2025-10-07T10:01:57.519057+00:00, map_index=3, execution_date=20251007T100157, start_date=20251007T101644, end_date=20251007T101652
[2025-10-07T10:16:52.557+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T10:16:52.585+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
